integration_tests/network.py

```diff
@@ -57,6 +112,39 @@ def setup_cronos(path, base_port):
         proc.wait()
 
 
+def setup_chainmain(path, base_port):
+    print(f"setup chainmain path= {path}")
```

Comment:
remove the duplicated print, the same information is printed in the next one.

Reply:
ok